tests/integration/test_render.py

```diff
@@ -194,3 +199,36 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_hidden(self):
```

Comment:
@agaylard To make it immediately obvious what's being tested you could change the names of the new tests to start with `test_zone_`. This can also be an advantage when having to debug tests (you'd know right away that it's *zone* borders that are causing trouble).